app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
```

Comment:
- [ ] Please modify the sign_in view in devise to match the wireframe and its styling.
## Wireframe Guidelines
![image](https://user-images.githubusercontent.com/46839548/200141111-b5479235-651b-4854-8845-61036ec1dfe2.png)
## Your Implementation
![image](https://user-images.githubusercontent.com/46839548/200141297-bdc0dab4-fb39-48bc-9b5a-6e5c1b5c6af3.png)